src/System.Security.Cryptography.Primitives/src/System/Security/Cryptography/CryptoStream.cs

```diff
@@ -271,12 +272,17 @@ private void CheckReadArguments(byte[] buffer, int offset, int count)
                     bytesToDeliver -= _outputBufferIndex;
                     currentOutputIndex += _outputBufferIndex;
                     _outputBufferIndex = 0;
+                    CryptographicOperations.ZeroMemory(_outputBuffer);
```

Comment:
Could we zero only through _outputBufferIndex?  Or we don't know if there's any meaningful bytes beyond that and so to be safe we clear the whole thing?